astropy/tests/runner.py

```diff
@@ -251,7 +251,7 @@ def run_tests(self, **kwargs):
         # also harmless to nest the contexts
         with set_temp_config(astropy_config, delete=True):
             with set_temp_cache(astropy_cache, delete=True):
-                return pytest.main(args=args, plugins=plugins)
+                retv = pytest.main(args=args, plugins=plugins)
```

Comment:
Is this an unrelated bug?

Reply:
Why is retv captured but not returned?

Reply:
whoops, that's a (broken) left over from my previous attempt, sorry (and thanks for catching it) !